### ISSUE TITLE
UI: Popup

### ISSUE THREAD

**Issue by lazzystep** [Panenco/panenco-ui#64]
##### But we can improve it a bit, here are my advices:
- When we open modal using keyboard and then when closing it, ideally we shouldn't lose focus and return it to button
- Add these attr's to docs and code:
 ```
 role="dialog"
 aria-labelledby="dialog4_label" (it's should be added by dev, would be nice nice to put it in docs)
 aria-describedby="dialog4_desc" (it's optional, can added by dev, would be nice to put it in docs)
 aria-modal="true"
 ```
